pkg/infrastructure/gcp/clusterapi/clusterapi.go

```diff
@@ -220,14 +220,10 @@ func (p Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
 }
 
 // DestroyBootstrap destroys the temporary bootstrap resources.
-func (p Provider) DestroyBootstrap(dir string) error {
+func (p Provider) DestroyBootstrap(ctx context.Context, in clusterapi.BootstrapDestroyInput) error {
 	logrus.Warnf("Destroying GCP Bootstrap Resources")
```

Comment:
As part of https://github.com/openshift/installer/pull/8359, in AWS, DestroyBootstrap() removedSSHRule. Do we need to do a similar thing here? 
Also, in AWS does DestroyBootstrap() have to similarly destroy storage for ignition based on preserveBootstrapIgnition/bestEffortDeleteIgnition?

/cc @r4f4 , @patrickdillon

Reply:
@sadasu ignition objects are automatically deleted by CAPA when the instance is provisioned.

Reply:
But maybe we should delete the bucket itself.

Reply:
Bug created: https://issues.redhat.com/browse/OCPBUGS-35037

Reply:
@barbacbd , @bfournie does GCP create a security group rule for SSH to the bootstrap?  https://github.com/openshift/installer/pull/8359 added to code to delete that rule during DestroyBootstrap() for AWS. Need something similar for GCP?

Reply:
Yes https://github.com/openshift/installer/pull/8374 added the rule.
The PR https://github.com/openshift/installer/pull/8523 will delete the rule, it could only be done after https://github.com/openshift/installer/pull/8359 merged